README.md

```diff
@@ -18,64 +17,75 @@ Then, add one or more of the modules below:
 
 The project consists of three modules that are independent of each other, comprising together a complete solution.
 
-### 1. Serializer
+### 1. Checker Plugin
 
-Simple Borer based Akka serializer. It uses codecs, provided by Borer, that are generated during compile time (so
-serializer won't crash during runtime like reflection-based serializers may do).
+A Scala compiler plugin that detects messages, events etc. and checks, whether they extend the base trait and reports
+and error when it doesn't. This ensures that the specified serializer is used by Akka and protects against accidental
+use of [Java serialization](https://doc.akka.io/docs/akka/current/serialization.html#java-serialization).
+
+To use, just annotate a base trait with `@org.virtuslab.ash.SerializabilityTrait`:
 
 ```scala
-libraryDependencies += repo %% "borer-akka-serializer" % commit
+@SerializabilityTrait
+trait MySerializable
+
 ```
 
-It may also be worth including additional codecs for common types that are missing in Borer standard library:
+Installation:
 
 ```scala
-libraryDependencies += repo %% "borer-extra-codecs" % commit
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" %% "serializability-checker-library" % commit
+libraryDependencies += compilerPlugin("com.github.VirtusLab.akka-serialization-helper" %% "serializability-checker-compiler-plugin" % commit)
 ```
 
-### 2. Checker Plugin
+### 2. Dump Event Schema
 
-A Scala compiler plugin that detects messages, events etc. and checks, whether they extend the base trait and reports
-and error when it doesn't. This ensures that the specified serializer is used by Akka and protects against accidental use
-of Java serialization.
+An sbt plugin that allows for dumping schema
+of [akka-persistence](https://doc.akka.io/docs/akka/current/typed/persistence.html#example-and-core-api) events to a
+file. Can be used for detecting accidental changes of events.
+
+Installation:
 
-To use, just annotate a base trait with `@org.virtuslab.akkasaferserializer.SerializabilityTrait`:
+Add to `plugins.sbt`:
 
 ```scala
-@SerializabilityTrait
-trait MySerializable
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" % "sbt-dump-event-schema" % commit
 ```
 
-Installation:
+To enable the plugin for a specific project, use:
 
 ```scala
-libraryDependencies += repo %% "akka-serializability-checker-library" % commit
-libraryDependencies += compilerPlugin(repo %% "akka-serializability-checker-plugin" % commit)
+enablePlugins(DumpEventSchemaPlugin)
 ```
 
-### 3. Dump Schema
+You also have to change one setting responsible for resolving companion compiler plugin:
 
-An sbt plugin that allows for dumping schema of events to a file. Can be used for detecting accidental changes of
-events.
+```scala
+dumpEventSchema / dumpEventSchemaCompilerPlugin := "com.github.VirtusLab.akka-serialization-helper" % "dump-event-schema-compiler-plugin" % commit 
```

Comment:
Hmm would it be possible that this compiler plugin is automatically resolved to jitpack and not to a local snapshot? Or will it only be reasonably doable once we move to Maven Central? (curious how it's handled by sbt plugins in general)

Reply:
Not really, that would require an sbt plugin to know how was it resolved

Reply:
Ok... I suppose that's the answer to first part of the question (jitpack)? or to both questions (jitpack + maven centrl)?

Reply:
With maven I solve this with default value:
```scala
dumpEventSchema / dumpEventSchemaCompilerPlugin := "org.virtuslab" %% "dump-event-schema-compiler-plugin" % "0.1.0-SNAPSHOT"
```

Reply:
Ok, that's what I suspected... I guess with Maven we'd need to somehow make sure that the default value for `dumpEventSchemaCompilerPlugin` will point to same version of the compiler plugin as the version of sbt plugin itself... as we rather don't want to maintain two independent versioning schemes for sbt plugin and compiler plugin :/